### PR TITLE
Nginx whitelist and clouflareProxy

### PR DESCRIPTION
**Pull request by mrkhachaturov**
Please check Makefile.

I can`t generate new readme.md

### CODE REVIEW

**Comment by mrkhachaturov** [comment]
[Please check Makefile. I can`t generate new readme.md](https://github.com/aenix-io/cozystack/pull/211)

**Review by kvaps — COMMENTED** [review]
Hi @mrkhachaturov thank you for your contribution! Could you please specify values in values.yaml file

Also, I'd like to change the naming for the options to be more generic, eg:
 
```yaml
## @param whitelist List of client networks
## Example:
## whitelist:
## - "1.2.3.4"
## - "10.100.0.0/16"

whitelist: []

## @param realIPProvider Enable configuration for specific RealIP provider. Allowed values: `none` and `cloudflare`
realIPProvider : "none"
```

**Comment by kvaps** [comment]
As about generiting values schema, it can be done using bitnamis [readme-generator-for-helm](https://github.com/bitnami/readme-generator-for-helm)

You can check the example here:
https://github.com/aenix-io/cozystack/blob/bbd2ca81a376d0ddc32a9f7264f7cdb53dc36b89/packages/apps/tcp-balancer/Makefile#L2-L3

File: packages/extra/ingress/Makefile [hunk 1]
```diff
@@ -1,2 +1,12 @@
+get-cloudflare-ips:
+	curl -s https://www.cloudflare.com/ips-v4/# | grep -oE '([0-9]{1,3}\.){3}[0-9]{1,3}/[0-9]{1,2}' > ips-v4.tmp
```
Comment (themoriarti) [inline packages/extra/ingress/Makefile]: I would recommend adding a timeout, for example 2-4 seconds, so as not to wait 30 or more seconds if this resource is unavailable for one reason or another.

**Review by themoriarti — COMMENTED** [review]
In general, it is an interesting functionality, but it would be good to describe it in a readme.

**Review by kvaps — APPROVED** [review]
Just fixed it a bit. LGTM

**Pull request merged**